Userland/Libraries/LibGUI/Window.cpp

```diff
@@ -522,6 +533,25 @@ void Window::propagate_shortcuts(KeyEvent& event, Widget* widget, ShortcutPropag
     event.ignore();
 }
 
+void Window::remember_state(DeprecatedString const& domain, DeprecatedString const& group)
+{
+    m_remember_state = true;
+    m_remember_state_domain = domain;
+    m_remember_state_group = group;
+
+    // Try to read window state from config
```

Comment:
This comment doesn't really add anything, it's clear that we're reading from the config just from the code.